packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -286,19 +281,17 @@ const resolveFluid = (image, options) => {
 
   // Add the original image (if it isn't already in there) to ensure the largest image possible
   // is available for small images.
+  const pwidth = parseInt(width, 10)
   if (
-    !filteredSizes.includes(parseInt(width)) &&
-    parseInt(width) < CONTENTFUL_IMAGE_MAX_SIZE &&
-    Math.round(width / desiredAspectRatio) < CONTENTFUL_IMAGE_MAX_SIZE
+    !filteredSizes.includes(parseInt(pwidth, 10)) &&
+    pwidth < CONTENTFUL_IMAGE_MAX_SIZE &&
+    Math.round(pwidth / desiredAspectRatio) < CONTENTFUL_IMAGE_MAX_SIZE
   ) {
-    filteredSizes.push(width)
+    filteredSizes.push(pwidth)
   }
 
-  // Sort sizes for prettiness.
-  const sortedSizes = _.sortBy(filteredSizes)
```

Comment:
This sort is similar to above except the condition before it does not exist in the other case.

The condition will only add the current width if it doesn't already exist here. And the loop before will cull any widths that are greater than this width. So either way, the width should be the last element in the list. This means the `.sortBy` is equally useless here and is safe to drop.